src/components/pages/Quote.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
```

Comment:
- Kindly, you disabled some functionality of the linter, :warning: the linter Lint checks against best practices, code style guideline violations, and structural problems. so please delete line number 1-3 :+1: